saddle-core/src/main/scala/org/saddle/Series.scala

```diff
@@ -925,12 +926,12 @@ class Series[X: ST: ORD, @spec(Int, Long, Double) T: ST](
     */
   def proxyWith(
       proxy: Series[X, T]
-  )(implicit fn: org.saddle.scalar.NA.type => T): Series[X, T] = {
```

Comment:
what's the issue here?

Reply:
Indeed, this can be reverted.


The implicit conversion (application of `fn`) does not trigger.